.github/ISSUE_TEMPLATE.md

```diff
@@ -14,31 +21,36 @@ in the next 30 days, the ticket will be automaticaly closed.
 Please describe your issue in English.
 -->
 
----------------------------------------------------
-BUG REPORT INFORMATION
----------------------------------------------------
+<h1>BUG REPORT INFORMATION </h1>
 
-**Centreon Web version**: 
+<h3> Prerequisites </h3>
 
-**Centreon Engine version**: 
+***Versions***
 
-**Centreon Broker version**: 
+-- Copy/Paste the result of the following command --
+```
+$ rpm -qa | grep centreon
```

Comment:
This is only valid for RPM based systems.

We need components versions also on unpackaged systems (like Debian).

Reply:
Fixed.